docs/facebook-example.rst

```diff
@@ -194,7 +194,106 @@ These are called *connections* in Girder Worker parlance.
         'output_step': 'find_neighborhood'}
    ]
 
-We now have a complete workflow! Let's run this, and write the final data to a file.
+We now have a complete workflow! For completeness, here is the full workflow specification as pure JSON.
+This file could be loaded with Python's ``json`` package and directly sent to ``girder_worker.run()``.
+
+.. code-block:: json
```

Comment:
Instead of copying the code, what about doing a literal include of json ?  Ideally, that same json would be used in a test (either a real test or its validation using a json-ld schema)
See http://www.sphinx-doc.org/en/stable/markup/code.html